android/src/main/java/com/twiliorn/library/niceday/NDExtra.java

```diff
@@ -0,0 +1,215 @@
+package com.twiliorn.library.niceday;
+
+import static com.twiliorn.library.niceday.NDHelper.parseDimensionsString;
+import static com.twiliorn.library.niceday.NDHelper.parsePriorityString;
+
+import android.util.Log;
+
+import androidx.annotation.Nullable;
+
+import com.facebook.react.bridge.ReadableMap;
+import com.twilio.video.BandwidthProfileMode;
+import com.twilio.video.BandwidthProfileOptions;
+import com.twilio.video.ConnectOptions;
+import com.twilio.video.EncodingParameters;
+import com.twilio.video.RemoteParticipant;
+import com.twilio.video.RemoteVideoTrack;
+import com.twilio.video.RemoteVideoTrackPublication;
+import com.twilio.video.Room;
+import com.twilio.video.TrackPriority;
+import com.twilio.video.TrackSwitchOffMode;
+import com.twilio.video.VideoBandwidthProfileOptions;
+import com.twilio.video.VideoDimensions;
+
+import java.util.HashMap;
+import java.util.Map;
+
+
+public class NDExtra {
```

Comment:
It would be nice if we add a bit of documentation explaining what does this NDExtra brings to the table. Or is that explained or detailed somewhere else? It looks like it's a "helper" so CustomTwilioVideoView.java is more clean -and therefore closer to the upstream version- or are there some other reasons?

Reply:
> It looks like it's a "helper" so CustomTwilioVideoView.java is more clean -and therefore closer to the upstream version- or are there some other reasons?

yes, i moved most of our changes on separate package `niceday`

Reply:
@jagamypriera Can you respond to this?

Reply:
Can we add a few lines on top of the class, or at least a minor explanation of each one of the methods?